index.html

```diff
@@ -248,8 +256,8 @@ <h2 class="contact-heading">Contact me</h2>
             <input class="contact-input" name="name" placeholder="Leslie" type="text" maxlength="30" required>
             <input class="contact-input" name="email" type="email" placeholder="leeegd99@gmail.com" required>
             <textarea class="textarea-input" name="textarea" cols="30" rows="9"
```

Comment:
[OPTIONAL] I suggest you change the place holders to `name` and `youremail@gmail.com`. Generalising it this way ensures that you increase its visibility plus avoid confusion especially for screen readers.